packages/apps/transfer/package.json

```diff
@@ -21,37 +24,25 @@
       "sort-package-json"
     ]
   },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
-  },
-  "eslintConfig": {
-    "extends": [
-      "react-app",
-      "react-app/jest"
-    ]
-  },
   "dependencies": {
     "@kadena/client": "workspace:*",
     "@kadena/pactjs": "workspace:*",
     "@kadena/pactjs-cli": "workspace:*",
     "@kadena/react-components": "workspace:*",
+    "@radix-ui/colors": "~0.1.8",
+    "@stitches/react": "~1.2.8",
     "@types/jest": "^29.5.1",
-    "@types/node": "^16.18.27",
+    "@types/node": "^18.16.0",
     "@types/react": "^18.2.6",
     "@types/react-dom": "^18.2.4",
+    "eslint": "8.41.0",
+    "eslint-config-next": "13.4.3",
+    "next": "13.4.3",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-router-dom": "^6.11.1",
-    "react-scripts": "5.0.1"
+    "react-scripts": "5.0.1",
+    "typescript": "5.0.4"
   },
   "devDependencies": {
```

Comment:
Can you add the package `sort-package-json` here as well? It's part of the lint-staged script but missing still. You can find the version used in the kadena docs project.